bin/webpack-entries.js

```diff
@@ -66,6 +66,9 @@ const blocks = {
 	'single-product': {
 		isExperimental: true,
 	},
+	'single-product-details': {
+		isExperimental: true,
```

Comment:
I think that it is not necessary. We want to ship this block.

Reply:
Great! I was waiting for the confirmation if we would make any design changes before shipping this block, but I'll remove this flag 🙌. Thank you!